docker-compose.yml

```diff
@@ -9,32 +9,16 @@ services:
       - .:/usr/src/app
     working_dir: '/usr/src/app'
     command: [rails, server, -p, '3000', -b, 0.0.0.0]
-    links:
+    depends_on:
       - postgres
     environment:
       - PG_USERNAME=postgres
       - PG_HOST=postgres
       - DIAPER_PARTNER_URL=http://diaperpartner:3000
       - DIAPER_PARTNER_SECRET_KEY=secretkey123
 
-  test:
```

Comment:
Having a separate block for `test` that was almost identical isn't necessary, especially since most of the time the specific test command is passed in anyway.